components/places/src/places.udl

```diff
@@ -45,6 +51,21 @@ interface PlacesConnection {
 
     [Throws=PlacesError]
     void metadata_delete_older_than(i64 older_than);
+
+    [Throws=PlacesError]
+    PublicNode? bookmarks_get_tree([ByRef] Guid item_guid);
+
+    [Throws=PlacesError]
+    PublicNode? bookmarks_get_by_guid([ByRef] Guid guid, u8 get_direct_children);
```

Comment:
as mentioned above, make this bool. Alternatively, lean in to it being an integer on the path to #4620 - rename the param to something like "fetchDepth", and for now, just `log::warn` if the param is > 1? Up to you :)